From: Sebastian Ramacher <sramacher@debian.org>
Date: Tue, 31 Aug 2021 08:53:52 +0200
Subject: Relax ABI check

We have symbol files and version dependencies to properly track this.
---
 vpx/src/vpx_decoder.c | 2 +-
 vpx/src/vpx_encoder.c | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/vpx/src/vpx_decoder.c b/vpx/src/vpx_decoder.c
index 427cd1b..eddbf1a 100644
--- a/vpx/src/vpx_decoder.c
+++ b/vpx/src/vpx_decoder.c
@@ -27,7 +27,7 @@ vpx_codec_err_t vpx_codec_dec_init_ver(vpx_codec_ctx_t *ctx,
                                        vpx_codec_flags_t flags, int ver) {
   vpx_codec_err_t res;
 
-  if (ver != VPX_DECODER_ABI_VERSION)
+  if (ver > VPX_DECODER_ABI_VERSION)
     res = VPX_CODEC_ABI_MISMATCH;
   else if (!ctx || !iface)
     res = VPX_CODEC_INVALID_PARAM;
diff --git a/vpx/src/vpx_encoder.c b/vpx/src/vpx_encoder.c
index f636b54..9f8eac9 100644
--- a/vpx/src/vpx_encoder.c
+++ b/vpx/src/vpx_encoder.c
@@ -32,7 +32,7 @@ vpx_codec_err_t vpx_codec_enc_init_ver(vpx_codec_ctx_t *ctx,
                                        vpx_codec_flags_t flags, int ver) {
   vpx_codec_err_t res;
 
-  if (ver != VPX_ENCODER_ABI_VERSION)
+  if (ver > VPX_ENCODER_ABI_VERSION)
     res = VPX_CODEC_ABI_MISMATCH;
   else if (!ctx || !iface || !cfg)
     res = VPX_CODEC_INVALID_PARAM;
@@ -67,7 +67,7 @@ vpx_codec_err_t vpx_codec_enc_init_multi_ver(
     int num_enc, vpx_codec_flags_t flags, vpx_rational_t *dsf, int ver) {
   vpx_codec_err_t res = VPX_CODEC_OK;
 
-  if (ver != VPX_ENCODER_ABI_VERSION)
+  if (ver > VPX_ENCODER_ABI_VERSION)
     res = VPX_CODEC_ABI_MISMATCH;
   else if (!ctx || !iface || !cfg || (num_enc > 16 || num_enc < 1))
     res = VPX_CODEC_INVALID_PARAM;
